app/controllers/admin/facility_groups_controller.rb

```diff
@@ -51,7 +42,7 @@ def destroy
   private
 
   def set_organizations
-    @organizations = policy_scope([:manage, :facility, Organization])
+    @organizations = policy_scope([:upcoming, :manage, Organization])
```

Comment:
Here is an alternate way we could write this filter which would apply to all controller actions:

```ruby
def require_organizations
  @organizations = current_admin.managed_organizations
  if @organizations.empty?
    render :not_authorized
  end
end
```

Reply:
I don't think we can apply `require_organizations` blindly to all controllers since it'll depend on what page you are at and what `managed_organizations` actually means. See this [comment](https://github.com/simpledotorg/simple-server/pull/1137#discussion_r459891772).

Reply:
I meant just this controller's actions.  Not _all_ controllers. I agree it depends on the context 100% !